Sources/CombineHelpers/ReplaySubject.swift

```diff
@@ -44,7 +44,7 @@ public final class ReplaySubject<Output, Failure: Error>: Subject {
     subscription.replay(self.buffer, completion: self.completion)
   }
 
-  private final class Subscription<Output, Failure: Error>: Combine.Subscription {
+  private final class Subscription: Combine.Subscription {
```

Comment:
This was a Swift warning and I don't think the shadowed generics were necessary.